DeDRM_plugin/ineptepub.py

```diff
@@ -518,32 +294,21 @@ def decryptBook(userkey, inpath, outpath):
                 return 1
 
             if len(bookkey) != 64:
-                # Normal Adobe ADEPT
-                rsa = RSA(userkey)
-                bookkey = rsa.decrypt(base64.b64decode(bookkey.encode('ascii')))
-
-                # Verify key:
-                if len(bookkey) > 16:
-                    # Padded as per RSAES-PKCS1-v1_5
-                    if verify_book_key(bookkey):
```

Comment:
Same comment as above - I'm not sure if removing `verify_book_key` is a good idea.